src/app/redux/reducers.ts

```diff
@@ -0,0 +1,32 @@
+import { ADD_ITEM, REMOVE_ITEM } from './actions';
```

Comment:
Given that you are already setting up your reducers in your slice.. Why are they again configured here? what's the use and what's the difference?